docs/developers/serializers.rst

```diff
@@ -0,0 +1,187 @@
+.. _serializers:
+
+Serializers
+===========
+
+You can customize the serialization to and from Stores in Pootle.
+
+This allows you to make formatting or other changes before saving to disk, and
+conversely you can make changes to files before importing into Pootle.
+
+Serializers and deserializers are configured for a project. See the
+`django-admin`:"project_serializers" command for more information on configuring projects.
+
+To add your own serializers of deserializers you must add your own django application,
+and include it in your `INSTALLED_APPS` setting.
+
+This document explains what files you will need in your application to add serializers
+or deserializers.
+
+
+.. _serializers#custom-serializer:
+
+Adding a custom serializer
+--------------------------
+
+Lets add a `CustomSerializer` in a file called `serializers.py`.
+
+The serializer class will be called with 2 arguments, `store`, and
+`original_data`.
+
+The original data is a bare representation of the Store according to the `Project`'s
+`localfiletype`, unless the data has already been through another serialization filter
+already.
```

Comment:
ill change to default - its diff for po/xliff ie they use diff ttk classes iirc, not sur about other Project.localfiletypes